scripts/infra

```diff
@@ -41,12 +58,34 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
                               -backend-config="key=terraform/${DB_STATE}/state"
                     terraform plan \
                               -var-file="${DB_SETTINGS_BUCKET}.tfvars" \
-                              -var="districtbuilder_image_version=\"${IMAGE_VERSION}\"" \
                               -var="state=\"${DB_STATE}\"" \
                               -out="${DB_SETTINGS_BUCKET}.tfplan"
                     ;;
                 apply)
+
                     terraform apply "${DB_SETTINGS_BUCKET}.tfplan"
+
+                    pushd "${PROJECT_ROOT}"
+
+                    echo "Configuring TLS for Docker..."
+                    aws s3 cp "s3://${DB_SETTINGS_BUCKET}/terraform/${DB_STATE}/.env" "${PROJECT_ROOT}/.env"
+
+                    mkdir -p "/root/.docker"
+                    aws s3 cp "s3://${DB_SETTINGS_BUCKET}/docker_certs/client/client.zip" "client.zip"
+                    unzip -od "/root/.docker" "client.zip"
+                    rm "client.zip"
+
+                    export DOCKER_TLS_VERIFY=1
+                    export DOCKER_HOST="origin.${DB_STATE}.districtbuilder.azavea.com:2476"
+                    export DOCKER_CERT_PATH="/root/.docker"
+
+                    set +e
+                    wait_for_docker
```

Comment:
Is this mostly to cover the edge case where a new machine is coming online and we're account for the Docker daemon restarting?

Reply:
That's correct.